packages/components/pie-link/src/link.scss

```diff
@@ -1,5 +1,17 @@
 @use '@justeattakeaway/pie-css/scss' as p;
 
+
+@mixin link-interactive-states($mode: 'default') {
+    &:hover,
```

Comment:
Do we need focus states here?

Reply:
are you referring to the focus visible? if so, it's below